components/scream/util/scream_catch_main.cpp

```diff
@@ -0,0 +1,16 @@
+#define CATCH_CONFIG_RUNNER
+
+#include "catch2/catch.hpp"
+
+#include "scream_kokkos.hpp"
+#include "scream_util.hpp"
+
+int main (int argc, char **argv) {
+  scream::util::activate_floating_point_exceptions_if_enabled();
```

Comment:
Make a scream_initialize and scream_finalize. Move the Kokkos, as well as the FPE, calls into it.